JobConfig/mixing/epilog.fcl

```diff
@@ -7,7 +7,7 @@ physics.filters.NeutralsFlashMixer.mu2e.simStageEfficiencyTags: ["NeutralsCat",
 physics.filters.MuStopPileupMixer.mu2e.simStageEfficiencyTags: ["MuBeamCat", "MuminusStopsCat", "MuStopPileupCat"]
 physics.filters.MuStopPileupMixer.mu2e.meanEventsPerPOTFactors : [ @local::Pileup.MuminusTargetStopPrescale ]
 # adjust detstep lists to include pileup
-physics.producers.makeSD.StrawGasStepModule : ""
+physics.producers.makeSD.StrawGasStepModules : [ "EleBeamFlashMixer", "MuBeamFlashMixer", "NeutralsFlashMixer", "MuStopPileupMixer" ]
```

Comment:
This list is the same as @sequence::Mixing.StepMixSequence, please use that instead.
I think you still need compressDetStepMCs in the list, otherwise you will miss the DetectorSteps coming from the primary particle (main event stream).

Reply:
Both components are addressed in the latest push, and validation in `Offline` was successful.